impeller/entity/entity_pass.h

```diff
@@ -142,7 +142,16 @@ class EntityPass {
   void SetEnableOffscreenCheckerboard(bool enabled);
 
   //----------------------------------------------------------------------------
-  /// @brief  Get the coverage of an unfiltered subpass.
+  /// @brief  Computes the coverage of a given subpass. This is used to
```

Comment:
Can you please include the coordinate space of the returned rect?  I think it should be relative to it's parent subpass in pixels?

Reply:
Done.

> I think it should be relative to it's parent subpass in pixels?

Actually, the output is in screen space. :)

After writing https://github.com/flutter/engine/pull/46524#issuecomment-1750194345, I also decided to do another iteration on the `GetCoverage` docstrings.

Reply:
Thank you! 🙏